css/src/components/scroll.scss

```diff
@@ -0,0 +1,11 @@
+.scroll-horizontal {
```

Comment:
Should we have a doc page presenting these classes? I think might be useful, otherwise, people won't know about their existence

Reply:
updated and added 👍